src/app/public/modules/select-field/select-field-picker-context.ts

```diff
@@ -11,7 +11,15 @@ import {
 } from './types/select-field-select-mode';
 
 export class SkySelectFieldPickerContext {
+  /**
+   * An array of `SkySelectField` objects from the select field's `data` property.
```

Comment:
I added these property descriptions because we have them in the current docs at https://developer.blackbaud.com/skyux/components/select-field#skyselectfieldpickercontext-properties, but they aren't displaying in the updated docs. Should they? And if so, what do we need to do to make that happen?

Reply:
@Blackbaud-SteveBrush, did I implement something wrong here? Or should I create a `docs-tools` issue for the fact that is not generating a table for the `SkySelectFieldPickerContext` properties?

Reply:
I updated the `docs-tools` version, and that fixed the missing `SkySelectFieldPickerContext` table, but it either exposed a new problem or I missed it the last time around. Now we have some HTML appearing in a `SkySelectFieldCustomPicker` table that I don't think should be there. We are exposing `code` and `a` tags that I don't think should appear in the docs, and this is causing the *Property* column to be overly wide, which pushes the *Description* column to the left, hiding the description and forcing consumers to use a scrollbar if they notice. Here's a screenshot of what I'm referring to:
![image](https://user-images.githubusercontent.com/7850105/91048644-f04de980-e5e9-11ea-926c-c44326325ca6.png)
Do we need to create another `docs-tools` issue for this? And if so, should we merge this PR as is or hold off for that issue?

Reply:
I'll work on a fix.

Reply:
Should we hold off on this PR until then? Or should we file a `docs-tools` issue and merge the PR?

Reply:
Fix has been approved/merged. I just need to release it. One minute.

Reply:
I upgraded `docs-tools` to 4.3.2, and it fixed one instance but on the other:
![image](https://user-images.githubusercontent.com/7850105/91315957-49528480-e786-11ea-8d72-fc839976ed8e.png)

Reply:
Released (yet) another fix, pushed the change, and I think we're good.